cpython-windows/build.py

```diff
@@ -421,6 +421,13 @@ def hack_props(
             rb"<tcltkDir>%s\$(ArchName)\</tcltkDir>" % tcltk_path,
         )
 
+    try:
+        static_replace_in_file(
+            tcltkprops_path, rb"<tclZlibDLLName>zlib1.dll</tclZlibDLLName>", rb""
+        )
+    except NoSearchStringError:
+        pass
```

Comment:
This part is slightly concerning.

Does CPython's build of tcl require a `zlib1.dll`? If so, removing this reference may mean we're not shipping a `zlib1.dll` requiring dependency file?

I don't think we have any references to a `zlib1.dll` in this project. If there is a `zlib1.dll` dependency here, that would be concerning.

Reply:
I don't think this patch (https://github.com/indygreg/python-build-standalone/pull/313/commits/97b944863bac338eff82c4b8e61a3bb8ac1a7fa0) actually accomplished anything. I'm just poking around here.

The upstream pull request upgrading the version (https://github.com/python/cpython/pull/101307) says:

> I don't see an easy way to avoid having a second copy of zlib specifically for Tcl, so I guess we just have to include it, but I'm pretty sure that's all handled here.

I'm not sure how to make sure this is linked to our zlib. The Windows distribution validation is failing with with:

```
error: python/install/DLLs/tcl86t.dll loads illegal library zlib1.dll
error: python/install/tcl/nmake/x86_64-w64-mingw32-nmakehlp.exe loads illegal library msvcrt.dll
```